cg/meta/transfer/lims.py

```diff
@@ -69,10 +69,13 @@ def __init__(self, status: Store, lims: LimsAPI):
     def _get_all_samples_not_yet_delivered(self):
         return self.status.samples_not_delivered()
 
-    def transfer_samples(self, status_type: SampleState, include="unset"):
+    def transfer_samples(self, status_type: SampleState, include="unset", sample_id=None):
```

Comment:
```suggestion
    def transfer_samples(self, status_type: SampleState, include:str = "unset", sample_id: str=None):
```
Let's try to use type hints as much as possible since it makes the code easier to read